src/power_grid_model_io/converters/tabular_converter.py

```diff
@@ -376,9 +407,15 @@ def _parse_col_def_const(data: TabularData, table: str, col_def: Union[int, floa
 
         """
         assert isinstance(col_def, (int, float))
-        return pd.DataFrame([col_def] * len(data[table]))
-
-    def _parse_col_def_column_name(self, data: TabularData, table: str, col_def: str) -> pd.DataFrame:
+        const_df = pd.DataFrame([col_def] * len(data[table]))
+        if table_mask is not None:
```

Comment:
Not sure we need to double check if `np.sum(table_mask)>0` here

Reply:
This reminds me to check the =0 condition. Ill do so.